plugins/inputs/lotus/lotus.go

```diff
@@ -383,6 +396,37 @@ func recordTipsetMessagesPoints(ctx context.Context, api api.FullNode, acc teleg
 	return nil
 }
 
+func recordLotusInfoPoints(ctx context.Context, api api.FullNode, acc telegraf.Accumulator, receivedAt time.Time, receivedFrom peer.ID) error {
+	version, err := api.Version(ctx)
+	if err != nil {
+		return err
+	}
+
+	network, err := api.StateNetworkName(ctx)
+	if err != nil {
+		return err
+	}
+	versionTokens := strings.Split(version.Version, "+")
+	if len(versionTokens) != 2 {
+		return fmt.Errorf("developer error, version string has changed format")
+	}
+
+	acc.AddFields("lotus_info",
+		map[string]interface{}{
+			"recorded_at": receivedAt.UnixNano(),
+		},
+		map[string]string{
+			"api_version": version.APIVersion.String(),
+			"block_delay": fmt.Sprintf("%d", version.BlockDelay),
+			"commit":      versionTokens[1],
+			"network":     string(network),
+			"peer_id":     receivedFrom.String(),
```

Comment:
Nit: `receivedFrom` doesn't really make sense in this context. Let's just call the var `peerID`.